pom.xml

```diff
@@ -76,21 +76,6 @@
             <artifactId>azure-identity</artifactId>
```

Comment:
I don't think that azure identity is used anymore, unless the cosmosclient needs it?

Reply:
good catch, removing it